cli/api.py

```diff
@@ -197,10 +197,10 @@ def generate_api_v2(model_output_dir, output, level, state, fips):
     us_timeseries = combined_datasets.load_us_timeseries_dataset()
 
     # Load all API Regions
-    regions = list(
-        us_timeseries.iter_regions(level=level, exclude_county_999=True, state=state, fips=fips)
+    selected_dataset = combined_datasets.load_us_timeseries_dataset().get_subset(
```

Comment:
us_timeseries is already loaded above, can you use that?

Reply:
removed extra local in d07d90cd